tests/certification/bindings/postgres/postgres_test.go

```diff
@@ -0,0 +1,241 @@
+	"testing"
+	"time"
+
+	_ "github.com/lib/pq"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/dapr/components-contrib/bindings"
+	binding_postgres "github.com/dapr/components-contrib/bindings/postgres"
+	bindings_loader "github.com/dapr/dapr/pkg/components/bindings"
+	"github.com/dapr/dapr/pkg/runtime"
+	dapr_testing "github.com/dapr/dapr/pkg/testing"
+	daprClient "github.com/dapr/go-sdk/client"
+	"github.com/dapr/kit/logger"
+
+	"github.com/dapr/components-contrib/tests/certification/embedded"
+	"github.com/dapr/components-contrib/tests/certification/flow"
+	"github.com/dapr/components-contrib/tests/certification/flow/dockercompose"
+	"github.com/dapr/components-contrib/tests/certification/flow/network"
+	"github.com/dapr/components-contrib/tests/certification/flow/sidecar"
+)
+
+const (
+	dockerComposeYAML      = "docker-compose.yml"
+	numOfMessages          = 10
+	dockerConnectionString = "host=localhost user=postgres password=example port=5432 connect_timeout=10 database=dapr_test sslmode=disable"
+)
+
+func TestPostgres(t *testing.T) {
+
+	ports, _ := dapr_testing.GetFreePorts(3)
+	grpcPort := ports[0]
+	httpPort := ports[1]
+
+	testExec := func(ctx flow.Context) error {
+		client, err := daprClient.NewClientWithPort(fmt.Sprintf("%d", grpcPort))
+		require.NoError(t, err, "Could not initialize dapr client.")
+
+		metadata := make(map[string]string)
+
+		ctx.Log("Invoking output binding for exec operation!")
+		req := &daprClient.InvokeBindingRequest{Name: "standard-binding", Operation: "exec", Metadata: metadata}
+		req.Metadata["sql"] = "INSERT INTO dapr_test_table (id, c1, ts) VALUES (1, 'demo', '2020-09-24T11:45:05Z07:00');"
+		errBinding := client.InvokeOutputBinding(ctx, req)
+		require.NoError(ctx, errBinding, "error in output binding - exec")
+
+		return nil
+	}
+
+	testQuery := func(ctx flow.Context) error {
+		client, err := daprClient.NewClientWithPort(fmt.Sprintf("%d", grpcPort))
+		require.NoError(t, err, "Could not initialize dapr client.")
+
+		metadata := make(map[string]string)
+
+		ctx.Log("Invoking output binding for query operation!")
+		req := &daprClient.InvokeBindingRequest{Name: "standard-binding", Operation: "query", Metadata: metadata}
+		req.Metadata["sql"] = "SELECT * FROM dapr_test_table WHERE id = 1;"
+		errBinding := client.InvokeOutputBinding(ctx, req)
+		require.NoError(ctx, errBinding, "error in output binding - query")
+
+		return nil
+	}
+
+	testClose := func(ctx flow.Context) error {
+		client, err := daprClient.NewClientWithPort(fmt.Sprintf("%d", grpcPort))
+		require.NoError(t, err, "Could not initialize dapr client.")
+
+		metadata := make(map[string]string)
+
+		ctx.Log("Invoking output binding for query operation!")
+		req := &daprClient.InvokeBindingRequest{Name: "standard-binding", Operation: "close", Metadata: metadata}
+		errBinding := client.InvokeOutputBinding(ctx, req)
+		require.NoError(ctx, errBinding, "error in output binding - close")
```

Comment:
Close is supposed to close the DB connection.

Please add a query operation here and assert the error -- the query should not work because the DB connection should be closed!